pkg/controller/networkpolicy/clusternetworkpolicy.go

```diff
@@ -137,16 +138,20 @@ func (n *NetworkPolicyController) deleteCNP(old interface{}) {
 
 // toAntreaServicesForCRD converts a secv1alpha1.NetworkPolicyPort object to an
 // Antrea Service object.
```

Comment:
mention the boolean parameter in the function comment?

Reply:
done